src/client/Microsoft.Identity.Client/Instance/Authority.cs

```diff
@@ -36,22 +27,65 @@ protected Authority(AuthorityInfo authorityInfo)
         public AuthorityInfo AuthorityInfo { get; }
 
         #region Builders
-        public static Authority CreateAuthorityWithOverride(AuthorityInfo requestAuthorityInfo, AuthorityInfo configAuthorityInfo)
+
+        /// <summary>
+        /// Figures out the authority based on the authority from the config and the authority from the request, 
+        /// and optionally the homeAccountTenantId, which has an impact on AcquireTokenSilent
+        /// 
+        /// The algorithm is: 
+        /// 
+        /// 1. If there is no request authority (i.e. no authority override), use the config authority. 
```

Comment:
>request [](start = 30, length = 7)

nit: the meaning of "request" authority is not very clear here, esp from a customer perspective. is it the authority used in the request?

Reply:
This is not a public comment, the class is internal. Request Authority is the same as "authority override"  - it is the authority that is specified in the AcquireToken* builder, e.g. 

` AcquireTokenSilent(...).WithAuthority(...)`

THis is different from the config authority, which is the one specified on the client object: 

`var pca = PublicClientApplicationBuilder.Create("clientid").WithAuthority(...)`

Reply:
Happy to rephrase in a way that makes more sense, please advise.